.github/workflows/_integration_tests.yaml

```diff
@@ -122,10 +122,12 @@ jobs:
           # Set enterprise to 'true' to enable isolated integration test cases requiring enterprise features.
           - name: isolated-dbless
             test: isolated.dbless
+            router-flavor: traditional
```

Comment:
If we do not set it here, it falls back to `expressions` defined in `TEST_KONG_ROUTER_FLAVOR` env above.